### DISCUSSION TITLE
Ability to remove recipients from the suppression list

### DISCUSSION THREAD

**Discussion by CHfish-ch** [postalserver/postal#1497] [Feature suggestions]
After reading #60 I still consider it necessary to have the option to remove address from the suppression list.
Allow me to explain: In our case Postal added multiple address to the suppression list because the of a malformed header (leading to the fact that the emails weren't accepted by the receiving server).
So retrying to send the messages that hard failed will not help (the header is still malformed).
And as of now we don't want to send the affected users any email, but we want to ensure that any intentioned email that might be sent at any time from now on is delivered.
This can only be done by either manually monitor messages being held or (more intelligent) removing the addresses from the suppression list.
Thank you for considering this.

**Comment by willpower232** [comment]
As it stands, you would have to send new emails to all the addresses, so that Postal holds the messages, and then force each one to be sent to clear them off the list.

Depending on the number of email addresses, it may be easier for you to clear them from the database first and then send out the new messages.

I'll leave this on feature request in case the others have any feedback or someone feels like making a PR.
